R/epidist_db.R

```diff
@@ -1,9 +1,12 @@
-#' Create an `<epidist>` object(s) directly from the epiparameter library
+#' Create `<epidist>` object(s) directly from the epiparameter library
 #' (database)
 #'
-#' @description Extract an `<epidist>` object(s) directly from
-#' the library of epidemiological parameters. This bypasses the need to
-#' read in an `<epiparam>` object and convert to an `<epidist>` object.
+#' @description Extract `<epidist>` object(s) directly from
+#' the library of epidemiological parameters. The epiparameter
+#' library of epidemiological parameters is compiled from primary literature
+#' sources. The list output from [epidist_db()] can be subset by the data it
+#' contains, for example by: disease, pathogen, epidemiological distribution,
+#' sample size, region, etc.
```

Comment:
Suggest listing everything.

Reply:
It is not easy to list every possible subsetting, since the function is set up to allow flexible subsetting with the `subset` argument (similar to how the NSE with the `subset()` function works). Therefore, it is possible to subset on most elements of an `<epidist>` object, of which there are quite a few. I'm happy to reword the documentation to make it clearer if you suggest an alternative.